src/main/java/com/Employes/Employedetail/controller/Employecontroller.java

```diff
@@ -0,0 +1,66 @@
+package com.Employes.Employedetail.controller;
+
+import com.Employes.Employedetail.Entity.Designation;
+import com.Employes.Employedetail.Entity.Employee;
+import com.Employes.Employedetail.repository.DesignationRepo;
+import com.Employes.Employedetail.repository.Employerepo;
+import com.Employes.Employedetail.service.employeeservice;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import sun.security.krb5.internal.crypto.Des;
+
+import javax.validation.constraints.Null;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/rest")
+public class Employecontroller {
+    @Autowired
+    Employerepo repo;
+    @Autowired
+    DesignationRepo depo;
+    @Autowired
+    employeeservice servo;
```

Comment:
Class name should be CamelCasing, starting with a Caps letter. So Class name should have been EmployeeService.